stage/vpc/waf.tf

```diff
@@ -8,27 +8,138 @@ resource "aws_wafv2_web_acl" "stage-default" {
   }
 
   rule {
-    name     = "stageRateLimitingRule"
+    name     = "stageRateLimitAuthenticated"
     priority = 1
 
     action {
       block {}
     }
 
+    statement {
+      rate_based_statement {
+        limit              = 3000
```

Comment:
For staging can this just be the same for all of them, like 1000, well if you want to do testing then feel free to leave it for now, but staging really should have a low rate limit as it's not meant for public consumption.